### DISCUSSION TITLE
เรามาใช้ SwiftUI กัน 🤩

### DISCUSSION THREAD

**Discussion by zkan** [dataength/daily#3] [Ideas]
ลุย @gatukgl

**Comment by zkan** [comment]
หาตัวอย่างแอพมือถือที่เอาไว้อ่านข่าวมากัน เช่น แบบนี้ https://www.pinterest.com/pin/714594665863614300/visual-search/

อยากได้แบบ cleanๆ simpleๆ อ่านเล่นระหว่างเดินทาง ไม่ต้องใช้เวลามาก อ่านผ่านๆ ก็พอได้ไอเดีย

**Comment by zkan** [comment]
Persona เบื้องต้น คือ เพิ่งจบใหม่ และก็ยังอยู่ในวัยทำงาน สนใจเรื่องของ Data Engineering แต่ไม่ค่อยมีเวลา เป็นคนไม่ชอบอ่านบทความยาวๆ เพราะต้องใช้สมาธิสูง เป็นคนที่เดินทางโดยรถสาธารณะ เช่น BTS/MRT

**Comment by gatukgl** [comment]
@zkan test

**Comment by gatukgl** [comment]
https://www.pinterest.com/pin/460282024425984771/

https://www.pinterest.com/pin/460282024425985200/

**Comment by zkan** [comment]
![Screen Shot 2564-09-10 at 21 05 18](https://user-images.githubusercontent.com/704794/132866171-074d3418-e5df-4719-957a-696773186b2e.png)

อันนี้กะเป็นหน้าแรก แล้วเลื่อนๆ ลงมาเรียงบทความ

**Comment by zkan** [comment]
![Screen Shot 2564-09-10 at 21 08 03](https://user-images.githubusercontent.com/704794/132866559-bdbf29a4-f49f-4861-9448-a0da4cc1c246.png)

![Screen Shot 2564-09-10 at 21 08 11](https://user-images.githubusercontent.com/704794/132866583-f4c0f57b-08a9-40c0-8cca-6fdf5ca14075.png)

อันนี้มี list แล้วกดไปอ่าน detail

**Reply by zkan** [reply]
ตกลงว่าจะลองทำแบบนี้กันก่อน เย้
